ms_frontend/root/src/js/login_register.js

```diff
@@ -88,7 +88,7 @@ function loginUserButton() {
     const email = 'marie.a.mensing@gmail.com'
     const enable2FA = true
 
-    const url = `${window.location.origin}/user/login/`
+    const url = `${window.location.origin}/user/login`
```

Comment:
@cryptoclown132 all endpoints going to the backend should not have a trailing slash